modules/analytics/lib/templates/plugin.js

```diff
@@ -0,0 +1,58 @@
+import Vue from 'vue';
+
+function isAnalyticsOn(ctx) {
+  let cookies = null
+  if (ctx.req != null) {
+    //Server side rendering
+    cookies = ctx.req.headers.cookie;
+  } else {
+    // Rely on the client
+    cookies = document.cookie;
+  }
+  let processed = {}
+  cookies.split(';').forEach((e) => {
+    let val = e.trim().split('=');
+    processed[val[0]] = decodeURI(val[1]);
+  })
+  let scopes = decodeURIComponent(processed['modrinth-scopes']).split(",");
+  return (scopes !== null && scopes.includes('analytics'));
```

Comment:
I would use `nuxt-cookies-universal` for this. It's a lot cleaner and the auth already uses it.

Reply:
Unfortunatly, it's in a module, so dependencies are not allowed there